uo_sys_framework/Core/cfgAttributes/3DEN/ExportLoadoutToClipboardAttribute.hpp

```diff
@@ -3,9 +3,9 @@ class UO_FW_ExportLoadoutToClipboardAttribute: Title {
         class Title: Title {};
         class Value: ctrlButtonPictureKeepAspect {
             idc=100;
-            x="(48) * (pixelW * pixelGrid *     0.50)";
-            w="0.99 *     5 * (pixelW * pixelGrid *     0.50)";
-            h="5 * (pixelH * pixelGrid *     0.50)";
+            x="(48) * (pixelW * pixelGrid * 0.50)";
```

Comment:
brackets not needed "48 * (pixelW * pixelGrid * 0.50)". There are other entries in the file like this.